src/sgame/sg_bot_ai.cpp

```diff
@@ -746,21 +746,25 @@ AINodeStatus_t BotActionFight( gentity_t *self, AIGenericNode_t *node )
 
 			if ( myTeam == TEAM_HUMANS )
 			{
-				if ( self->botMind->botSkill.level >= 3 && DistanceToGoalSquared( self ) < Square( MAX_HUMAN_DANCE_DIST )
```

Comment:
This change seems less readable to me since it creates variables named "dist" which are actually squared distances.